siriuspy/siriuspy/devices/pwrsupply.py

```diff
@@ -203,6 +203,13 @@ def pwrstate(self, value):
         """."""
         self['PwrState-Sel'] = value
 
+    def set_strength(self, value, tol=0.2, timeout=10, wait_mon=False):
+        """Set RF phase and wait until it gets there."""
+        self.strength = value
+        pv2wait = self._strength_rb_propty if wait_mon \
```

Comment:
Shouldn't be `pv2wait = self._strength_mon_propty if wait_mon else self._strength_rb_propty`?

Reply:
Right. Done!